common/schemes/wc_stasis_trap_scheme.txt

```diff
@@ -0,0 +1,181 @@
+﻿stasis_trap_scheme = {
+	skill = intrigue
+	hostile = yes
+
+	power_per_skill_point = 2.0
+	resistance_per_skill_point = 1.75
+	power_per_agent_skill_point = 1
+	
+	minimum_progress_chance = 10
+	maximum_progress_chance = 90
+	minimum_success = 5
+	maximum_secrecy = 85
+	maximum_success = 85
```

Comment:
I would consider changing these stats. Murder has a bigger maximum success and it's faster.

Reply:
Changed it to the murder ones, should maybe be even more lenient

Reply:
> Changed it to the murder ones, should maybe be even more lenient

But its has less impactful result than murder. If its like murder, I would do murder instead.

Reply:
Changed them to be even more lenient, tell me if you'd like it to be even easier

Reply:
> Changed them to be even more lenient, tell me if you'd like it to be even easier

 LGTM!